ipaplatform/redhat/services.py

```diff
@@ -45,8 +45,8 @@
 redhat_system_units = dict((x, "%s.service" % x)
                            for x in base_services.wellknownservices)
 
-redhat_system_units['rpcgssd'] = 'nfs-secure.service'
-redhat_system_units['rpcidmapd'] = 'nfs-idmap.service'
+redhat_system_units['rpcgssd'] = 'rpc-gssd.service'
+redhat_system_units['rpcidmapd'] = 'nfs-idmapd.service'
```

Comment:
Debian/Ubuntu has ``rpc-gssd.service`` but the ID mapper service is still called ``nfs-idmapd.service``. Please update the Debian service definition, too.

Reply:
Debian already has in ipaplatform/debian/services.py:
~~~
debian_system_units['rpcgssd'] = 'rpc-gssd.service'
debian_system_units['rpcidmapd'] = 'nfs-idmapd.service'
~~~